packages/amplify-e2e-tests/src/rds-v2-tests-common/rds-model-v2.ts

```diff
@@ -154,7 +145,6 @@ export const testRDSModel = (engine: ImportedRDSType, queries: string[]): void =
       expect(rdsPatchingSubscription.Properties.Protocol).toEqual('lambda');
       expect(rdsPatchingSubscription.Properties.Endpoint).toBeDefined();
       expect(rdsPatchingSubscription.Properties.TopicArn).toBeDefined();
-      expect(rdsPatchingSubscription.Properties.TopicArn).toMatchObject(expectedTopicArn);
```

Comment:
Can we at least do a partial match here?

Reply:
Added a check for the value here.